packages/frontend/src/components/SchedulersView/SchedulersView.tsx

```diff
@@ -0,0 +1,378 @@
+import {
+    getHumanReadableCronExpression,
+    isSlackTarget,
+    SchedulerJobStatus,
+    SchedulerWithLogs,
+} from '@lightdash/common';
+import {
+    Anchor,
+    Box,
+    Group,
+    MantineTheme,
+    Stack,
+    Table,
+    Text,
+    Tooltip,
+} from '@mantine/core';
+import {
+    IconAlertTriangleFilled,
+    IconCircleCheckFilled,
+    IconClockFilled,
+    IconCsv,
+    IconPhoto,
+    IconProgress,
+} from '@tabler/icons-react';
+import React, { FC, useMemo } from 'react';
+import { useTableStyles } from '../../hooks/styles/useTableStyles';
+import MantineIcon from '../common/MantineIcon';
+import { IconBox } from '../common/ResourceIcon';
+import SchedulersViewActionMenu from './SchedulersViewActionMenu';
+
+export type SchedulerItem = SchedulerWithLogs['schedulers'][number];
+export type Log = SchedulerWithLogs['logs'][number];
+
+type ColumnName =
+    | 'name'
+    | 'destinations'
+    | 'frequency'
+    | 'lastDelivery'
+    | 'actions'
+    | 'jobs'
+    | 'deliveryScheduled'
+    | 'deliveryStarted'
+    | 'status';
+
+type SchedulersProps = {
+    projectUuid: string;
+    schedulers: SchedulerWithLogs['schedulers'];
+    logs: SchedulerWithLogs['logs'];
+    users: SchedulerWithLogs['users'];
+    charts: SchedulerWithLogs['charts'];
+    dashboards: SchedulerWithLogs['dashboards'];
+};
+
+export interface Column {
+    id: ColumnName;
+    label?: string;
+    cell: (item: SchedulerItem) => React.ReactNode;
+    meta?: {
+        style: React.CSSProperties;
+    };
+}
+
+export const getSchedulerIcon = (item: SchedulerItem, theme: MantineTheme) => {
+    switch (item.format) {
+        case 'csv':
+            return (
+                <IconBox
+                    icon={IconCsv}
+                    color="indigo.6"
+                    style={{ color: theme.colors.indigo[6] }}
+                />
+            );
+        case 'image':
+            return (
+                <IconBox
+                    icon={IconPhoto}
+                    color="indigo.6"
+                    style={{ color: theme.colors.indigo[6] }}
+                />
+            );
+    }
+};
+
+export const getLogStatusIcon = (log: Log, theme: MantineTheme) => {
+    switch (log.status) {
+        case SchedulerJobStatus.SCHEDULED:
+            return (
+                <Tooltip label={SchedulerJobStatus.SCHEDULED}>
+                    <MantineIcon
+                        icon={IconClockFilled}
+                        color="blue.3"
```

Comment:
I feel like those functions should be extracted as separate utils, because they are being used in both views